fetch.bs

```diff
@@ -8495,6 +8624,98 @@ with a <var>promise</var>, <var>request</var>, <var>responseObject</var>, and an
 </div>
 
 
+<h3 id=fetch-later-method>FetchLater method</h3>
+
+<pre class=idl>
+
+dictionary DeferredRequestInit : RequestInit {
+  DOMHighResTimeStamp activationTimeout;
```

Comment:
I think this is a bit too strongly hinting that the fetch would happen this amount time later.
Could it be activationDeadline or some such to indicate that it could happen also earlier?

Reply:
Agreed, it's a deadline rather than a timeout.

Reply:
Note that it could also happen later. So deadline may not be appropriate.

Reply:
How about `delay`?